satellite/audit/verifier_test.go

```diff
@@ -313,23 +358,29 @@ func TestVerifierHappyPath(t *testing.T) {
 	testplanet.Run(t, testplanet.Config{
 		SatelliteCount: 1, StorageNodeCount: 4, UplinkCount: 1,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
-		audits := planet.Satellites[0].Audit.Service
-		err := audits.Close()
-		require.NoError(t, err)
+		planet.Satellites[0].Audit.Worker.Loop.Pause()
```

Comment:
nit - this pattern is followed in several tests I've seen, and it can be reordered:
```
satellite := planet.Satellites[0]
audits := satellite.Audit
queue := audits.Queue

audits.Worker.Loop.Pause()
```